zkevm-circuits/src/evm_circuit.rs

```diff
@@ -408,32 +409,56 @@ pub mod test {
     }
 
     pub fn get_test_degree<F: Field>(block: &Block<F>) -> u32 {
+        let num_rows_required_for_execution_steps: usize =
+            EvmCircuit::<F>::get_num_rows_required(block);
+        let num_rows_required_for_rw_table: usize = block.circuits_params.max_rws;
+        let num_rows_required_for_fixed_table: usize = detect_fixed_table_tags(block)
+            .iter()
+            .map(|tag| tag.build::<F>().count())
+            .sum();
+        let num_rows_required_for_bytecode_table: usize = block
+            .bytecodes
+            .values()
+            .map(|bytecode| bytecode.bytes.len())
```

Comment:
I noticed that this was previously incorrect, so maybe you could add the fix here: The bytecode circuit takes 1 extra row for each bytecode to encode its length, so this would be `.map(|bytecode| bytecode.bytes.len() + 1)`.

Note that this also means we shouldn't parametrize the Bytecode circuit with a `max_bytecode` parameter (I may be the culprit for that) because it's not accurate (due to the Length rows that don't contain any byte).

Reply:
fixed!